packages/backend/server/src/core/doc/reader.ts

```diff
@@ -178,9 +177,7 @@ export class DatabaseDocReader extends DocReader {
     }
     let avatarUrl: string | undefined;
     if (content.avatarKey) {
-      avatarUrl = this.url.link(
-        `/api/workspaces/${workspaceId}/blobs/${content.avatarKey}`
-      );
+      avatarUrl = this.blobStorage.getAvatarUrl(workspaceId, content.avatarKey);
```

Comment:
@forehalo The added storage dependency here is not actually using the functionality of r2, and there are currently no issues.